packages/firestore/test/unit/specs/spec_builder.ts

```diff
@@ -1029,7 +1029,7 @@ export class SpecBuilder {
   private addQueryToActiveTargets(
     targetId: number,
     query: Query,
-    resumeToken?: string
+    resumeToken?: string | TestSnapshotVersion
```

Comment:
Make this `resumeFrom` as well, since TestSnapshotVersion is not a resumeToken.

Reply:
Separated.